app.js

```diff
@@ -28,6 +28,10 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 const index = require('./routes/index');
 app.use('/', index);
 
```

Comment:
You need to import the session.config file, otherwise your session won't be stored